### PR TITLE
III-5244 - Fix unable to save an event with "locatie in overleg met de school" as location

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- [Use AudienceType const](https://github.com/cultuurnet/udb3-frontend/pull/556/commits/3acda49f8290f0cbb738f4affdab2c67940b7360)

### Changed
- Show that event is a cultuurkuur event based on the audienceType of the event

### Fixed
- unable to save an event with "locatie in overleg met de school" as location
- when selecting "locatie in overleg met de school" the audienceType is set to "everyone"

---
Ticket:
 - https://jira.uitdatabank.be/browse/III-5244 
 - https://jira.uitdatabank.be/browse/III-5246

### CODE REVIEW

File: src/pages/create/OfferForm.tsx [hunk 1]
```diff
@@ -138,6 +140,13 @@ const OfferForm = () => {
     return undefined;
   }, [asPath, query.scope]);
 
+  const offerId =
+    scope === OfferTypes.EVENTS && typeof query.eventId === 'string'
+      ? query.eventId
+      : scope === OfferTypes.PLACES && typeof query.placeId === 'string'
+      ? query.placeId
+      : undefined;
```
Comment (simon-debruijn) [inline src/pages/create/OfferForm.tsx:148]: You could rewrite it to this:

```suggestion
 const offerId = arrayToValue(scope === OfferTypes.EVENTS ? query.eventId : query.placeId) || undefined
```

Comment (brampauwelyn) [reply src/pages/create/OfferForm.tsx]: Done ✅

File: src/pages/create/OfferForm.tsx [hunk 2]
```diff
@@ -262,7 +281,10 @@ const OfferForm = () => {
         scopeStepConfiguration,
         typeAndThemeStepConfiguration,
         calendarStepConfiguration,
-        locationStepConfiguration,
+        {
+          ...locationStepConfiguration,
+          offerId,
+        },
```
Comment (simon-debruijn) [inline src/pages/create/OfferForm.tsx:287]: I think this isn't needed. offerId should be passed automatically to all steps in the configurations array

**Review by simon-debruijn — APPROVED** [review]

File: src/pages/create/OfferForm.tsx (hunk 2 above)
Comment (brampauwelyn) [reply src/pages/create/OfferForm.tsx]: Yes you're right :)

**Pull request merged**